frame/support/src/storage/migration.rs

```diff
@@ -24,128 +24,56 @@ use crate::hash::ReversibleStorageHasher;
 
 use super::PrefixIterator;
 
-/// Utility to iterate through raw items in storage.
-pub struct StorageIterator<T> {
-	prefix: Vec<u8>,
-	previous_key: Vec<u8>,
-	drain: bool,
-	_phantom: ::sp_std::marker::PhantomData<T>,
+/// Construct iterator to iterate over map items in `module` for the map called `item`.
```

Comment:
aren't changes to this file a bit unrelated? These have nothing to do with child trie afaik.

Reply:
Yes, but the linked issue also suggested that the `Storage[Key]Iterator`s can use `PrefixIterator` underneath, so I went ahead and made the change.

Reply:
maybe we should deprecated those types instead of removing them.

Reply:
Ahh okay. 👍